addon/mixins/node-actions.js

```diff
@@ -1,44 +1,61 @@
 import Ember from 'ember';
 
+/**
+ * @module ember-osf
+ * @submodule mixins
+ */
+
+/**
+ * Controller mixin that implements common actions performed on nodes.
+ * @class NodeActionsMixin
+ * @extends Ember.Mixin
+ */
 export default Ember.Mixin.create({
+    /**
+     * The node to perform these actions on. If not specified, defaults to the model hook.
+     * @property node
+     * @type DS.Model
+     */
     node: null,
     model: null,
     _node: Ember.computed.or('node', 'model'),
     /**
-    * Helper method that maps all node contributors to format {contribId: Contributor}
-    *
-    * @method _generateContributorMap
-    * @param {Array} contributors All contributor relationships on the node
-    * @return {Object} Returns a contributor map of the id to the contributor record
-    */
+     * Helper method that maps all node contributors to format {contribId: Contributor}
+     *
+     * @method _generateContributorMap
+     * @private
+     * @param {Array} contributors All contributor relationships on the node
```

Comment:
Could be `Contributor[]` instead of `Array`.

Might change the description to:
> A list of contributors to be mapped